Library/Formula/ps2eps.rb

```diff
@@ -1,18 +1,22 @@
-require 'formula'
-
 class Ps2eps < Formula
-  homepage 'http://www.tm.uka.de/~bless/ps2eps'
-  url 'http://www.tm.uka.de/~bless/ps2eps-1.68.tar.gz'
-  sha1 '253b949b70e73beb68760aafc5a20357db17371a'
-
-  depends_on 'ghostscript'
+  homepage "http://www.tm.uka.de/~bless/ps2eps"
+  url "http://www.tm.uka.de/~bless/ps2eps-1.68.tar.gz"
+  sha256 "b08f12eed88965d1891261fb70e87c7e3a3f3172ebc31bdb7994a7ce854dd925"
 
-  skip_clean 'bin' # Don't strip +x on ps2eps
+  depends_on "ghostscript"
 
   def install
-    system "#{ENV.cc} #{ENV.cflags} src/C/bbox.c -o bbox"
-    bin.install "bin/ps2eps", "bbox"
+    system ENV.cc, "src/C/bbox.c", "-o", "bbox"
+    bin.install "bbox"
+    # add missing shebang
+    (bin/"ps2eps").write("#!/usr/bin/env perl\n#{File.read("bin/ps2eps")}")
```

Comment:
Sent the email and not hearing back for a week. Maybe I should rewrite it as a wrapper script then merge it?